pages/primary-nav-pages/code-of-conduct/index.md

```diff
@@ -4,8 +4,39 @@ layout: page
 path: "/code-of-conduct/"
 priority: 1
 hide: true
+className: "faq-viewport"
 ---
 
-<h1>Code of Conduct</h1>
 
-#### PR Code of Conduct
+DEF CON 402 is commited to providing a harassment-free experience for every person, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, age, race, religion, or editor preference.  We do not tolerate any forms of harassment of group participants.
```

Comment:
Add ethnicity and national origin

Reply:
Typo. "commited" should be "committed". I would also add "DEF CON 402 is committed to providing **an inclusive and** harassment-free experience for every person..."

Reply:
I dunno...I mean we have to have some minimum standards... 
what if an emacs users shows up?